zk/src/archive/web/js/zk/fmt/msgfmt.js

```diff
@@ -13,7 +13,7 @@ This program is distributed under LGPL Version 2.1 in the hope that
 it will be useful, but WITHOUT ANY WARRANTY.
 */
 zk.fmt.Text = {
-	format: function (msg) {
+	stuff: function (msg) {
 		var i = 0, sb = '';
```

Comment:
Why naming "stuff"? it's hard to know what it can do

Reply:
ok